remix_01.py

```diff
@@ -39,11 +39,18 @@ def filter_json(id):
     user_name = data['user_name']
     bpm = data['upload_extra']['bpm']
     upload_name = data['upload_name']
+    remix_parents = []
```

Comment:
Instead of just getting the first `data['remix_parents'][0]` parent, we append (add) all the parents to a new list